src/react-hook/react-hook.tsx

```diff
@@ -19,11 +19,11 @@ export function useConfig() {
       "ModuleNameContext has not been provided. This should come from openmrs-react-root-decorator"
     );
   }
-  if (!config) {
+  if (typeof config === "undefined") {
```

Comment:
This typeof check looks unnecessary to me. You use `typeof` when you're referencing a variable that you're not sure even exists. In this case, we know that the `config` variable exists.

Are you worried about `config` being falsy somehow, but still need to go into this if statement? I don't see any possible way that that would be the case.

Reply:
Oop, I was mistaken about the right way to check if a variable has value `undefined` in JS.